### ISSUE TITLE
MNIST에 대한 실험

### ISSUE THREAD

**Issue by chatterboy** [chatterboy/sad#3]

**Comment by chatterboy** [comment]
1. MNIST 디렉토리 생성
2. MNIST 데이터셋 다운

**Comment by chatterboy** [comment]
[1]에서는 MNIST에 대한 실험결과를 다음과 같이 표로 나타냈다.
![image](https://user-images.githubusercontent.com/9608328/51460255-491a2800-1d9e-11e9-975c-da25b8695036.png)
**참고**
[1] N. Srivastava, et al. Dropout: A simple way to prevent neural networks from overfitting
